### PR TITLE
Add custom classes

### PR DESCRIPTION
**Pull request by VladimirJarabica**

### CODE REVIEW

File: src/Calendar.jsx [hunk 1]
```diff
@@ -308,13 +312,18 @@ export default class Calendar extends React.PureComponent<Props, State> {
...
+      customClasses
     } = this.props;
 
     const { currentMonth, hoveredDates, isFocused } = this.state;
 
     const mergedColors = R.merge(defaultColors, colors);
     const months = utils.getMonths(firstMonth, lastMonth);
+    const filteredCustomClasses = utils.filterCustomClasses(
+      allowedDates[0],
+      allowedDates[allowedDates.length - 1]
```
Comment (dominikchrastek) [inline src/Calendar.jsx]: u can use just head, last from ramda

File: example/Example.jsx [hunk 2]
```diff
@@ -77,6 +77,10 @@ export default class Example extends React.PureComponent<Props, State> {
           rangeSelect
           firstMonth={subMonths(new Date(), 1)}
           lastMonth={addMonths(new Date(), 1)}
+          customClasses={{
+            class1: [new Date()],
+            class2: [new Date(), addDays(new Date(), 2)],
```
Comment (dominikchrastek) [inline example/Example.jsx]: add also class1 and class2 with some style somewhere just for example

File: src/utils/index.js [hunk 3]
```diff
@@ -105,3 +108,23 @@ export const calendarMonthsToRender = (
   }
   return R.compose(R.take(visibleMonths), R.drop(currentMonth))(months);
 };
+
+type CustomClasses = { [className: string]: Date[] };
+
+export const filterCustomClasses = (
```
Comment (dominikchrastek) [inline src/utils/index.js]: add pls some comment what this all code do in general

File: example/Example.jsx (hunk 2 above)
Comment (VladimirJarabica) [reply example/Example.jsx]: ok

File: src/utils/index.js (hunk 3 above)
Comment (VladimirJarabica) [reply src/utils/index.js]: done 👊

**Pull request merged**